### ISSUE TITLE
Verified extension author should be part of aria label of extension items

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#175666]
Currently the aria label of extension items does not announce if the author is verified or not.
We should change that as it was requested by users and overall makes good sense.

fyi @meganrogge

**Comment by isidorn** [comment]
Awesome! Works great -> verified
